### PR TITLE
pkg/semtech-loramac: fix doxygen documentation

### PR DESCRIPTION
**Pull request by aabadie**
### Contribution description

This PR fixes the doxygen documentation of the Semtech Loramac package.

### Issues/PRs references

Related to #8664 and based on #8665

### CODE REVIEW

**Comment by aabadie** [comment]
Well, this PR is pretty urgent in fact. I need it for today (preparing practical work document that requires this). It's marked 'Waiting for other PR' (#8665) but it just does the same without excluding loramac pkg doc. @miri64 is it fine for you ?

**Comment by jia200x** [comment]
@aabadie it's fine for me, since the idea is to add the remaining packages to doxygen.

Just tested and works as expected.

**Comment by jia200x** [comment]
would be good to wait @miri64 and @gebart though

File: doc/doxygen/riot.doxyfile [hunk 1]
```diff
@@ -827,7 +827,10 @@ EXCLUDE_PATTERNS       = */board/*/tools/* \
                          */cpu/native/osx-libc-extra \
                          */cpu/x86/include/* \
                          */drivers/kw2xrf/include/overwrites.h \
-                         */pkg/*/*/* \
+                         */pkg/emb6/* \
+                         */pkg/lwip/* \
+                         */pkg/nordic_softdevice_ble/* \
+                         */pkg/openthread/* \
```
Comment (miri64) [inline doc/doxygen/riot.doxyfile]: Unrelated?

Comment (miri64) [reply doc/doxygen/riot.doxyfile]: Sorry, didn't noticed this belonged to #8665.

**Comment by miri64** [comment]
Please rebase.

**Comment by miri64** [comment]
(to current master)

**Comment by aabadie** [comment]
Apparently, it's not needed, the last master commit is also in this branch

**Comment by aabadie** [comment]
I rebased, just in case...

**Comment by miri64** [comment]
Yes, but it makes the changes in this PR clearer in the "Files changed" view and keeps the overall Git history cleaner:

```
master
|\
| PR
|/
|\
| PR
|/
.
.
.
```

instead of 

```
master
|\
| \
|  PR
|\/
| PR
|/

**Comment by aabadie** [comment]
Murdock is happy (even if he's very busy ;)).

**Comment by aabadie** [comment]
Thanks for reviewing @miri64 and @jia200x, the documentation is now ok :)
